src/controllers/confirm.company.controller.ts

```diff
@@ -0,0 +1,25 @@
+import { NextFunction, Request, Response } from "express";
+import { Templates } from "../model/template.paths";
+import { getValueFromObjectionsSession } from "../services/objections.session.service";
+import logger from "../utils/logger";
+
+/**
+ * GET controller for check company details screen
+ * @param req
+ * @param res
+ * @param next
+ */
+
+export const route = async (req: Request, res: Response, next: NextFunction): Promise<void> => {
+    if (req.session && req.session.data) {
+        const company: string = getValueFromObjectionsSession(req.session, "company_data");
```

Comment:
I think that rather than 'string' you can use 'ObjectionCompanyProfile' as the type here?

Reply:
I don't follow - the string is the key.

Reply:
Sorry, I mean 'const company: ObjectionCompanyProfile...'. Will that work? Stronger typing...

Reply:
Refactored this.